x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -1123,23 +1211,54 @@ public void findActiveTokensForUser(String username, ActionListener<Collection<T
                     )
                     .should(QueryBuilders.boolQuery()
                         .must(QueryBuilders.termQuery("refresh_token.invalidated", false))
-                        .must(QueryBuilders.rangeQuery("creation_time").gte(now.toEpochMilli() - TimeValue.timeValueHours(24).millis()))
+                        .must(QueryBuilders.rangeQuery("creation_time").gte(now.toEpochMilli()
+                                - TimeValue.timeValueHours(ExpiredTokenRemover.MAXIMUM_TOKEN_LIFETIME_HOURS).millis()))
                     )
                 );
-
-            final SearchRequest request = client.prepareSearch(SECURITY_INDEX_NAME)
+            final SearchRequest request = client.prepareSearch(tokensIndex.aliasName())
                 .setScroll(DEFAULT_KEEPALIVE_SETTING.get(settings))
                 .setQuery(boolQuery)
                 .setVersion(false)
                 .setSize(1000)
                 .setFetchSource(true)
                 .request();
-            securityIndex.checkIndexVersionThenExecute(listener::onFailure,
+            tokensIndex.checkIndexVersionThenExecute(listener::onFailure,
                 () -> ScrollHelper.fetchAllByEntity(client, request, listener,
                     (SearchHit hit) -> filterAndParseHit(hit, isOfUser(username))));
         }
     }
 
+    private BytesReference createTokenDocument(UserToken userToken, @Nullable String refreshToken,
+                                               @Nullable Authentication originatingClientAuth) {
+        assert refreshToken == null || originatingClientAuth != null;
```

Comment:
Can we have a message on this assert so we know the state of these parameters if it fails.

Reply:
The only failure state is `refreshToken` != `null` and `originatingClientAuth` == `null` . I've added a message to make this clear.